app/services/current-user.js

```diff
@@ -25,7 +27,8 @@ export default Ember.Service.extend({
         deferred.resolve(null);
       });
     } else {
-      this.get('store').find('user', currentUserId).then(function(user){
+      this.get('store').find('user', currentUserId).then((user) => {
+        this.set('user', user);
```

Comment:
`current-user` probably needs some refactoring. It should be called `session` or `authManager`. We need to set a `user` or `currentUser` property with user object that the server returns and store it there after a user logs in so we can retrieve any user related info on the spot without making any requests.